.github/workflows/lint.yml

```diff
@@ -47,3 +47,11 @@ jobs:
         python tools/lint.py --diff-against origin/$GITHUB_BASE_REF
         python tools/unicode-check.py
         python tools/check_test_name.py
+
+    - name: Check that Python.h is first in any file including it.
+      shell: bash
+      run: |
+        python tools/check_python_h_first.py $(find scipy \
+            \( -name highspy -o -name boost \) -prune \
+            -o -type f \( -iname \*.[ch] -o -name \*.[ch]pp -o -name \*.[ch]xx \
+                -o -name \*.cc -o -name \*.hh \) -print)
```

Comment:
Can this be written without the `$(find ...)`? Easier to understand, and it then matches `python dev.py lint`. The exclusions don't seem needed for me, but if they are then maybe move them into the defaults of the script?